### PR TITLE
fix effectstable

### PR DESCRIPTION
**Pull request by chinook25**

### CODE REVIEW

**Review by kzalite — APPROVED** [review]

File: app/ts/EffectsTable/EffectsTableCriteriaRows/EffectsTableDataSourceRow/ValueCell/EffectValueCell/EffectValueCellService.ts [hunk 1]
```diff
@@ -37,44 +42,65 @@ function renderEnteredValues(effect: Effect, usePercentage: boolean): string {
...
 
-function getStringForValue(value: number, usePercentage: boolean): string {
+function getStringForValue(
+  value: number,
+  usePercentage: boolean,
+  unitOfMeasurementType: UnitOfMeasurementType
+): string {
   if (usePercentage) {
-    return significantDigits(value * 100) + '%';
+    const modifer = unitOfMeasurementType === 'decimal' ? 100 : 1;
```
Comment (DanielReid) [inline app/ts/EffectsTable/EffectsTableCriteriaRows/EffectsTableDataSourceRow/ValueCell/EffectValueCell/EffectValueCellService.ts:51]: typo `modifer`

File: app/ts/EffectsTable/EffectsTableCriteriaRows/EffectsTableDataSourceRow/ValueCell/EffectValueCell/EffectValueCellService.ts [hunk 2]
```diff
@@ -37,44 +42,65 @@ function renderEnteredValues(effect: Effect, usePercentage: boolean): string {
   }
 }
 
-function getStringForValue(value: number, usePercentage: boolean): string {
```
Comment (DanielReid) [inline app/ts/EffectsTable/EffectsTableCriteriaRows/EffectsTableDataSourceRow/ValueCell/EffectValueCell/EffectValueCellService.ts:40]: kind of two-minded file with half the functions being 'get' without retrieving and the other half 'render' without creating DOM elements directly :D

File: shared/workspaceService.ts [hunk 3]
```diff
@@ -74,36 +75,39 @@ export function buildNewStyleCopy<T>(
...
-        dataSource.id,
-        dataSource.unitOfMeasurement.type === 'percentage'
-      ];
-    });
+    return buildDataSourcePercentageMap(criterion);
   });
   return _.fromPairs(values);
 }
 
+function buildDataSourcePercentageMap(
```
Comment (DanielReid) [inline shared/workspaceService.ts:100]: this function seems misnamed, I don't see any percentage.

**Review by DanielReid — COMMENTED** [review]
some small things

**Pull request merged**